### ISSUE TITLE
Hold Notes / Long Notes / LN

### ISSUE THREAD

**Issue by dustdustinthewind** [PulsarcGame/Pulsarc#25]
A controversial feature, but one that should be discussed and considered before it's ruled out or confirmed.

Edit: Please look at https://github.com/PulsarcGame/Pulsarc/issues/25#issuecomment-514496357 instead of the ugly Discord Reference Links below.

~~Here's a shit ton of Discord References:~~

~~https://discordapp.com/channels/600885985219444741/600886496677068825/601104058790707247~~

~~Traditional LNs will not look clean:~~
~~https://discordapp.com/channels/600885985219444741/600886496677068825/601225948968517633~~
~~https://discordapp.com/channels/600885985219444741/600886496677068825/601228435842531354~~
~~https://discordapp.com/channels/600885985219444741/600886496677068825/601231952917299210~~
~~https://discordapp.com/channels/600885985219444741/600886496677068825/601228226517401623~~

~~Judgement for release may need to be more lenient, or not judged at all:~~
~~https://discordapp.com/channels/600885985219444741/600886496677068825/601226776399708170~~

~~A stationary hold arc may be a good solution, similar to the Special Hold Notes in Cytus II where there is an indication of how much time you still need to hold:
https://discordapp.com/channels/600885985219444741/601713748683784192/601913353513992192~~

**Comment by jaanjah** [comment]
Perhaps send images of the messages instead of sending links to the discord, since if you aren't in the server you can't see the references.

I think the discussion about having LNs or not should've been resolved before making an issue about it though.

**Comment by dustdustinthewind** [comment]
Good point! Still trying to figure things out here. Thanks for all the help though!

Here's some screenshots of the messages, I'll work on that for all the other issues too, probably shouldn't have done that to begin with. Sorry for the inconvenience! 😅 

![image](https://user-images.githubusercontent.com/16842122/61769659-d830e880-ad9f-11e9-80ce-73f3805697f7.png)

![image](https://puu.sh/DUawX/5283484665.png)

![image](https://user-images.githubusercontent.com/16842122/61769719-1af2c080-ada0-11e9-8207-60c52acd0493.png)

![image](https://user-images.githubusercontent.com/16842122/61769785-51304000-ada0-11e9-9f94-072adcf1a36a.png)

**Comment by jaanjah** [comment]
I don't think the skinning/look of the LN matters too much, it can easily be changed later.

The thing that matters is how the LN would behave on screen, which we have to discuss how exactly playing a LN arc would look like. Will we animate it that you hold it and it continues going to the crosshair, or does it slowly remove the arc while holding it and etc.
